File: interview-app-objc/interview-app-objc/ListTableViewCell.m
//
//  ListTableViewCell.m
//  interview-app-objc
//
//  Created by Tony Nuzzi on 5/23/15.
//  Copyright (c) 2015 Tony Nuzzi. All rights reserved.
//

#import "ListTableViewCell.h"

@implementation ListTableViewCell

- (void)awakeFromNib {
    // Make image view round
    [self.image.layer setCornerRadius:25];
    [self.image.layer setMasksToBounds:YES];
}
@end

File: interview-app-swift/interview-app-swift/ListTableViewCell.swift
//
//  ListTableViewCell.swift
//  interview-app-swift
//
//  Created by Tony Nuzzi on 6/4/15.
//  Copyright (c) 2015 Tony Nuzzi. All rights reserved.
//

import UIKit

class ListTableViewCell: UITableViewCell {

    @IBOutlet weak var businessImage: UIImageView!
    @IBOutlet weak var name: UILabel!
    @IBOutlet weak var ratingImage: UIImageView!
    @IBOutlet weak var numOfReviews: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()

        // Make the business circle completely round
        businessImage.businessImageCircle()
    }
}